src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+import 'bulma/css/bulma.css';
+import '@fortawesome/fontawesome-free/css/all.css';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            className={cn({ 'is-active': tab.id === selectedTab.id })}
```

Comment:
would be better to move it to the variable & reuse it
```suggestion
            className={cn({ 'is-active': isActive })}
```